tests/unit/test_gff3_parser.py

```diff
@@ -0,0 +1,102 @@
+"""
+Data-driven tests for rna quantification.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import ga4gh.gff3 as gff3
+import unittest
+
+
+class TestGff3ParserOnTypicalFile(unittest.TestCase):
+    """
+    Data driven unit tests for the GFF3 parser
+    """
+    def setUp(self):
```

Comment:
FWIW, these really are meant to be unit tests of the parser. Known inputs, known desired parsed object outcome. As you said, @jeromekelleher, I'm just aiming for reasonable test coverage of the parser code - and since we're still evolving the parser code, I figured getting these tests in was a high priority.